vw_golf_mk4.dbc

```diff
@@ -1138,6 +1138,39 @@ BO_ 210 HCA_1: 5 XXX
  SG_ LM_OffSign : 31|1@1+ (1,0) [0|1] "" XXX
  SG_ Vib_Freq : 32|4@1+ (4,0) [0|60] "Hz" XXX
  SG_ Vib_Amp : 36|4@1+ (0.5,0) [0|7.5] "Nm" XXX
+ 
+BO_ 644 mMotor_Bremse: 6 Motor_Diesel
```

Comment:
```suggestion
BO_ 644 Motor_Bremse: 6 XXX
```

The prefixing `m` on message names is unique to the source of this data, we have been removing them.